### PR TITLE
terraform setup for redis

### PR DESCRIPTION
**Pull request by VinothRajasekar**
redis instance for databricks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# Include override files you do wish to add to version control using negated pattern
+#
+# !example_override.tf
+
+# Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
+# example: *tfplan*
+
+# Ignore generated tm-terraform backend info
+backend.tf
```
Comment (calistenson) [inline .gitignore:32]: Add new lines to the end of all of your files.

Comment (VinothRajasekar) [reply .gitignore]: sure. taken care.

**Comment by VinothRajasekar** [comment]
```Staging Plan

Login Succeeded
terraform {
  backend "s3" {
    bucket = "teikametrics-staging-terraform-state"
    key    = "databricks-redis/terraform//staging.tfstate"
    region = "us-east-1"
  }
}
tm-terraform 1.8 starting (sha 2742a596ebb44eb3a7689a9a49a45931f4c50fa8)
Initializing modules...
Downloading cloudposse/elasticache-redis/aws 0.40.0 for redis...
- redis in .terraform/modules/redis
Downloading cloudposse/route53-cluster-hostname/aws 0.12.0 for redis.dns...
- redis.dns in .terraform/modules/redis.dns
Downloading cloudposse/label/null 0.24.1 for redis.dns.this...
- redis.dns.this in .terraform/modules/redis.dns.this
Downloading cloudposse/security-group/aws 0.3.1 for redis.security_group...
- redis.security_group in .terraform/modules/redis.security_group
Downloading cloudposse/label/null 0.24.1 for redis.security_group.this...
- redis.security_group.this in .terraform/modules/redis.security_group.this
Downloading cloudposse/label/null 0.24.1 for redis.this...
- redis.this in .terraform/modules/redis.this
Downloading git@github.com:teikametrics/tm-terraform.git?ref=0.30 for redis_monitors...
- redis_monitors in .terraform/modules/redis_monitors/modules/aws/elasticache/redis

Initializing the backend...

Successfully configured the backend "s3"! Terraform will automatically
use this backend unless the backend configuration changes.

Initializing provider plugins...
- Finding hashicorp/null versions matching ">= 2.0.*"...
- Finding hashicorp/local versions matching ">= 1.2.*"...
- Finding hashicorp/aws versions matching ">= 2.0.*, >= 2.0.*, >= 2.0.*"...
- Finding latest version of datadog/datadog...
- Finding latest version of hashicorp/random...
- Using hashicorp/random v3.1.0 from the shared cache directory
- Using hashicorp/null v3.1.0 from the shared cache directory
- Using hashicorp/local v2.1.0 from the shared cache directory
- Using hashicorp/aws v3.73.0 from the shared cache directory
- Using datadog/datadog v3.8.1 from the shared cache directory

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, we recommend adding version constraints in a required_providers block
in your configuration, with the constraint strings suggested below.

* datadog/datadog: version = "~> 3.8.1"
* hashicorp/random: version = "~> 3.1.0"

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

module.redis_monitors.data.aws_ssm_parameter.datadog_app_key: Refreshing state...
module.redis_monitors.data.aws_ssm_parameter.datadog_api_key: Refreshing state...
data.aws_vpc.vpc: Refreshing state...
data.aws_subnet_ids.private: Refreshing state...

------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # random_password.password will be created
  + resource "random_password" "password" {
      + id          = (known after apply)
      + length      = 16
      + lower       = true
      + min_lower   = 0
      + min_numeric = 0
      + min_special = 0
      + min_upper   = 0
      + number      = true
      + result      = (sensitive value)
      + special     = false
      + upper       = true
    }

  # module.redis.aws_elasticache_parameter_group.default[0] will be created
  + resource "aws_elasticache_parameter_group" "default" {
      + arn         = (known after apply)
      + description = "Managed by Terraform"
      + family      = "redis6.x"
      + id          = (known after apply)
      + name        = "databricks-ai-redis-cluster"
      + tags_all    = (known after apply)
    }

  # module.redis.aws_elasticache_replication_group.default[0] will be created
  + resource "aws_elasticache_replication_group" "default" {
      + apply_immediately              = true
      + arn                            = (known after apply)
      + at_rest_encryption_enabled     = false
      + auth_token                     = (sensitive value)
      + auto_minor_version_upgrade     = true
      + automatic_failover_enabled     = false
      + cluster_enabled                = (known after apply)
      + configuration_endpoint_address = (known after apply)
      + data_tiering_enabled           = (known after apply)
      + engine                         = "redis"
      + engine_version                 = "6.x"
      + engine_version_actual          = (known after apply)
      + global_replication_group_id    = (known after apply)
      + id                             = (known after apply)
      + maintenance_window             = "sun:03:00-wed:04:00"
      + member_clusters                = (known after apply)
      + multi_az_enabled               = false
      + node_type                      = "cache.t4g.small"
      + number_cache_clusters          = 1
      + parameter_group_name           = "databricks-ai-redis-cluster"
      + port                           = 6379
      + primary_endpoint_address       = (known after apply)
      + reader_endpoint_address        = (known after apply)
      + replication_group_description  = "databricks-ai-redis-cluster"
      + replication_group_id           = "databricks-ai-redis-cluster"
      + security_group_ids             = (known after apply)
      + security_group_names           = (known after apply)
      + snapshot_retention_limit       = 7
      + snapshot_window                = "04:00-05:00"
      + subnet_group_name              = "databricks-ai-redis-cluster"
      + tags                           = {
          + "Name" = "databricks-ai-redis-cluster"
        }
      + tags_all                       = {
          + "Name" = "databricks-ai-redis-cluster"
        }
      + transit_encryption_enabled     = (known after apply)

      + cluster_mode {
          + num_node_groups         = (known after apply)
          + replicas_per_node_group = (known after apply)
        }
    }

  # module.redis.aws_elasticache_subnet_group.default[0] will be created
  + resource "aws_elasticache_subnet_group" "default" {
      + arn         = (known after apply)
      + description = "Managed by Terraform"
      + id          = (known after apply)
      + name        = "databricks-ai-redis-cluster"
      + subnet_ids  = [
          + "subnet-05409461b8b024a8b",
          + "subnet-07286da5b6a147c54",
          + "subnet-08544c8b5a0bd00c4",
        ]
      + tags_all    = (known after apply)
    }

  # module.redis_monitors.datadog_monitor.elasticache_cpu will be created
  + resource "datadog_monitor" "elasticache_cpu" {
      + evaluation_delay    = (known after apply)
      + id                  = (known after apply)
      + include_tags        = true
      + message             = <<~EOT
            Elasticache redis cluster databricks-ai-redis-cluster CPU utilization is critical.
            {{#is_no_data}}Not receiving data{{/is_no_data}}


            @slack-team-artificial-intelligence-alerts
        EOT
      + name                = "databricks-ai-redis-cluster cpu utilization critical."
      + new_host_delay      = 300
      + notify_no_data      = false
      + query               = "min(last_5m):max:aws.elasticache.cpuutilization{engine:redis,name:databricks-ai-redis-cluster} by {name} > 90"
      + require_full_window = true
      + tags                = [
          + "service:databricks-ai-redis-staging",
          + "stage:staging",
        ]
      + type                = "metric alert"
    }

  # module.redis_monitors.datadog_monitor.elasticache_curr_connections will be created
  + resource "datadog_monitor" "elasticache_curr_connections" {
      + evaluation_delay    = (known after apply)
      + id                  = (known after apply)
      + include_tags        = true
      + message             = <<~EOT
            Elasticache redis cluster databricks-ai-redis-cluster paralell connection use is critical.
            {{#is_no_data}}Not receiving data{{/is_no_data}}


            @slack-team-artificial-intelligence-alerts
        EOT
      + name                = "databricks-ai-redis-cluster parallel connection percentage critical."
      + new_host_delay      = 300
      + notify_no_data      = false
      + query               = "min(last_5m):max:aws.elasticache.curr_connections{engine:redis,name:databricks-ai-redis-cluster} by {name} > 58500"
      + require_full_window = true
      + tags                = [
          + "service:databricks-ai-redis-staging",
          + "stage:staging",
        ]
      + type                = "metric alert"
    }

  # module.redis_monitors.datadog_monitor.elasticache_memory_usage will be created
  + resource "datadog_monitor" "elasticache_memory_usage" {
      + evaluation_delay    = (known after apply)
      + id                  = (known after apply)
      + include_tags        = true
      + message             = <<~EOT
            Elasticache redis cluster databricks-ai-redis-cluster memory usage is critical.
            {{#is_no_data}}Not receiving data{{/is_no_data}}


            @slack-team-artificial-intelligence-alerts
        EOT
      + name                = "databricks-ai-redis-cluster database memory usage critical."
      + new_host_delay      = 300
      + notify_no_data      = false
      + query               = "min(last_5m):max:aws.elasticache.database_memory_usage_percentage{engine:redis,name:databricks-ai-redis-cluster} by {name} > 90"
      + require_full_window = true
      + tags                = [
          + "service:databricks-ai-redis-staging",
          + "stage:staging",
        ]
      + type                = "metric alert"
    }

  # module.redis.module.security_group.aws_security_group.default[0] will be created
  + resource "aws_security_group" "default" {
      + arn                    = (known after apply)
      + description            = "ElastiCache Security Group"
      + egress                 = (known after apply)
      + id                     = (known after apply)
      + ingress                = (known after apply)
      + name                   = "databricks-ai-redis-cluster"
      + name_prefix            = (known after apply)
      + owner_id               = (known after apply)
      + revoke_rules_on_delete = false
      + tags                   = {
          + "Name" = "databricks-ai-redis-cluster"
        }
      + tags_all               = {
          + "Name" = "databricks-ai-redis-cluster"
        }
      + vpc_id                 = "vpc-0ea0205e13be13fa7"
    }

  # module.redis.module.security_group.aws_security_group_rule.default["egress--1-0-0-9c87e5e1ed040a443ce1ac8e6d6cf159"] will be created
  + resource "aws_security_group_rule" "default" {
      + cidr_blocks              = [
          + "0.0.0.0/0",
        ]
      + description              = "Allow all outbound traffic"
      + from_port                = 0
      + id                       = (known after apply)
      + protocol                 = "-1"
      + security_group_id        = (known after apply)
      + self                     = false
      + source_security_group_id = (known after apply)
      + to_port                  = 0
      + type                     = "egress"
    }

  # module.redis.module.security_group.aws_security_group_rule.default["ingress-tcp-6379-6379-c992040fb21cf75967d80aa440691f00"] will be created
  + resource "aws_security_group_rule" "default" {
      + cidr_blocks              = [
          + "10.74.0.0/16",
          + "10.50.0.0/16",
        ]
      + description              = "Allow TCP inbound traffic on port 6379 from Convox and the vpc which is hosting this redis."
      + from_port                = 6379
      + id                       = (known after apply)
      + protocol                 = "tcp"
      + security_group_id        = (known after apply)
      + self                     = false
      + source_security_group_id = (known after apply)
      + to_port                  = 6379
      + type                     = "ingress"
    }

Plan: 10 to add, 0 to change, 0 to destroy.

**Comment by VinothRajasekar** [comment]
```production```

```
Login Succeeded
terraform {
  backend "s3" {
    bucket = "teikametrics-production-terraform-state"
    key    = "databricks-redis/terraform//production.tfstate"
    region = "us-east-1"
  }
}
tm-terraform 1.8 starting (sha 2742a596ebb44eb3a7689a9a49a45931f4c50fa8)
Initializing modules...
Downloading cloudposse/elasticache-redis/aws 0.40.0 for redis...
- redis in .terraform/modules/redis
Downloading cloudposse/route53-cluster-hostname/aws 0.12.0 for redis.dns...
- redis.dns in .terraform/modules/redis.dns
Downloading cloudposse/label/null 0.24.1 for redis.dns.this...
- redis.dns.this in .terraform/modules/redis.dns.this
Downloading cloudposse/security-group/aws 0.3.1 for redis.security_group...
- redis.security_group in .terraform/modules/redis.security_group
Downloading cloudposse/label/null 0.24.1 for redis.security_group.this...
- redis.security_group.this in .terraform/modules/redis.security_group.this
Downloading cloudposse/label/null 0.24.1 for redis.this...
- redis.this in .terraform/modules/redis.this
Downloading git@github.com:teikametrics/tm-terraform.git?ref=0.30 for redis_monitors...
- redis_monitors in .terraform/modules/redis_monitors/modules/aws/elasticache/redis

Initializing the backend...

Successfully configured the backend "s3"! Terraform will automatically
use this backend unless the backend configuration changes.

Initializing provider plugins...
- Finding hashicorp/aws versions matching ">= 2.0.*, >= 2.0.*, >= 2.0.*"...
- Finding latest version of datadog/datadog...
- Finding latest version of hashicorp/random...
- Finding hashicorp/null versions matching ">= 2.0.*"...
- Finding hashicorp/local versions matching ">= 1.2.*"...
- Using hashicorp/local v2.1.0 from the shared cache directory
- Using hashicorp/aws v3.73.0 from the shared cache directory
- Using datadog/datadog v3.8.1 from the shared cache directory
- Using hashicorp/random v3.1.0 from the shared cache directory
- Using hashicorp/null v3.1.0 from the shared cache directory

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, we recommend adding version constraints in a required_providers block
in your configuration, with the constraint strings suggested below.

* datadog/datadog: version = "~> 3.8.1"
* hashicorp/random: version = "~> 3.1.0"

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

data.aws_vpc.vpc: Refreshing state...
module.redis_monitors.data.aws_ssm_parameter.datadog_api_key: Refreshing state...
module.redis_monitors.data.aws_ssm_parameter.datadog_app_key: Refreshing state...
data.aws_subnet_ids.private: Refreshing state...

------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # random_password.password will be created
  + resource "random_password" "password" {
      + id          = (known after apply)
      + length      = 16
      + lower       = true
      + min_lower   = 0
      + min_numeric = 0
      + min_special = 0
      + min_upper   = 0
      + number      = true
      + result      = (sensitive value)
      + special     = false
      + upper       = true
    }

  # module.redis.aws_elasticache_parameter_group.default[0] will be created
  + resource "aws_elasticache_parameter_group" "default" {
      + arn         = (known after apply)
      + description = "Managed by Terraform"
      + family      = "redis6.x"
      + id          = (known after apply)
      + name        = "databricks-ai-redis-cluster"
      + tags_all    = (known after apply)
    }

  # module.redis.aws_elasticache_replication_group.default[0] will be created
  + resource "aws_elasticache_replication_group" "default" {
      + apply_immediately              = true
      + arn                            = (known after apply)
      + at_rest_encryption_enabled     = false
      + auth_token                     = (sensitive value)
      + auto_minor_version_upgrade     = true
      + automatic_failover_enabled     = false
      + cluster_enabled                = (known after apply)
      + configuration_endpoint_address = (known after apply)
      + data_tiering_enabled           = (known after apply)
      + engine                         = "redis"
      + engine_version                 = "6.x"
      + engine_version_actual          = (known after apply)
      + global_replication_group_id    = (known after apply)
      + id                             = (known after apply)
      + maintenance_window             = "sun:03:00-wed:04:00"
      + member_clusters                = (known after apply)
      + multi_az_enabled               = false
      + node_type                      = "cache.m6g.large"
      + number_cache_clusters          = 1
      + parameter_group_name           = "databricks-ai-redis-cluster"
      + port                           = 6379
      + primary_endpoint_address       = (known after apply)
      + reader_endpoint_address        = (known after apply)
      + replication_group_description  = "databricks-ai-redis-cluster"
      + replication_group_id           = "databricks-ai-redis-cluster"
      + security_group_ids             = (known after apply)
      + security_group_names           = (known after apply)
      + snapshot_retention_limit       = 7
      + snapshot_window                = "04:00-05:00"
      + subnet_group_name              = "databricks-ai-redis-cluster"
      + tags                           = {
          + "Name" = "databricks-ai-redis-cluster"
        }
      + tags_all                       = {
          + "Name" = "databricks-ai-redis-cluster"
        }
      + transit_encryption_enabled     = (known after apply)

      + cluster_mode {
          + num_node_groups         = (known after apply)
          + replicas_per_node_group = (known after apply)
        }
    }

  # module.redis.aws_elasticache_subnet_group.default[0] will be created
  + resource "aws_elasticache_subnet_group" "default" {
      + arn         = (known after apply)
      + description = "Managed by Terraform"
      + id          = (known after apply)
      + name        = "databricks-ai-redis-cluster"
      + subnet_ids  = [
          + "subnet-069fc5b52a45de0a2",
          + "subnet-0ac13e3f814c74d83",
          + "subnet-0fbf621a714c153b7",
        ]
      + tags_all    = (known after apply)
    }

  # module.redis_monitors.datadog_monitor.elasticache_cpu will be created
  + resource "datadog_monitor" "elasticache_cpu" {
      + evaluation_delay    = (known after apply)
      + id                  = (known after apply)
      + include_tags        = true
      + message             = <<~EOT
            Elasticache redis cluster databricks-ai-redis-cluster CPU utilization is critical.
            {{#is_no_data}}Not receiving data{{/is_no_data}}


            @slack-team-artificial-intelligence-alerts
        EOT
      + name                = "databricks-ai-redis-cluster cpu utilization critical."
      + new_host_delay      = 300
      + notify_no_data      = false
      + query               = "min(last_5m):max:aws.elasticache.cpuutilization{engine:redis,name:databricks-ai-redis-cluster} by {name} > 90"
      + require_full_window = true
      + tags                = [
          + "service:databricks-ai-redis-production",
          + "stage:production",
        ]
      + type                = "metric alert"
    }

  # module.redis_monitors.datadog_monitor.elasticache_curr_connections will be created
  + resource "datadog_monitor" "elasticache_curr_connections" {
      + evaluation_delay    = (known after apply)
      + id                  = (known after apply)
      + include_tags        = true
      + message             = <<~EOT
            Elasticache redis cluster databricks-ai-redis-cluster paralell connection use is critical.
            {{#is_no_data}}Not receiving data{{/is_no_data}}


            @slack-team-artificial-intelligence-alerts
        EOT
      + name                = "databricks-ai-redis-cluster parallel connection percentage critical."
      + new_host_delay      = 300
      + notify_no_data      = false
      + query               = "min(last_5m):max:aws.elasticache.curr_connections{engine:redis,name:databricks-ai-redis-cluster} by {name} > 58500"
      + require_full_window = true
      + tags                = [
          + "service:databricks-ai-redis-production",
          + "stage:production",
        ]
      + type                = "metric alert"
    }

  # module.redis_monitors.datadog_monitor.elasticache_memory_usage will be created
  + resource "datadog_monitor" "elasticache_memory_usage" {
      + evaluation_delay    = (known after apply)
      + id                  = (known after apply)
      + include_tags        = true
      + message             = <<~EOT
            Elasticache redis cluster databricks-ai-redis-cluster memory usage is critical.
            {{#is_no_data}}Not receiving data{{/is_no_data}}


            @slack-team-artificial-intelligence-alerts
        EOT
      + name                = "databricks-ai-redis-cluster database memory usage critical."
      + new_host_delay      = 300
      + notify_no_data      = false
      + query               = "min(last_5m):max:aws.elasticache.database_memory_usage_percentage{engine:redis,name:databricks-ai-redis-cluster} by {name} > 90"
      + require_full_window = true
      + tags                = [
          + "service:databricks-ai-redis-production",
          + "stage:production",
        ]
      + type                = "metric alert"
    }

  # module.redis.module.security_group.aws_security_group.default[0] will be created
  + resource "aws_security_group" "default" {
      + arn                    = (known after apply)
      + description            = "ElastiCache Security Group"
      + egress                 = (known after apply)
      + id                     = (known after apply)
      + ingress                = (known after apply)
      + name                   = "databricks-ai-redis-cluster"
      + name_prefix            = (known after apply)
      + owner_id               = (known after apply)
      + revoke_rules_on_delete = false
      + tags                   = {
          + "Name" = "databricks-ai-redis-cluster"
        }
      + tags_all               = {
          + "Name" = "databricks-ai-redis-cluster"
        }
      + vpc_id                 = "vpc-0134131a45fbf70e6"
    }

  # module.redis.module.security_group.aws_security_group_rule.default["egress--1-0-0-9c87e5e1ed040a443ce1ac8e6d6cf159"] will be created
  + resource "aws_security_group_rule" "default" {
      + cidr_blocks              = [
          + "0.0.0.0/0",
        ]
      + description              = "Allow all outbound traffic"
      + from_port                = 0
      + id                       = (known after apply)
      + protocol                 = "-1"
      + security_group_id        = (known after apply)
      + self                     = false
      + source_security_group_id = (known after apply)
      + to_port                  = 0
      + type                     = "egress"
    }

  # module.redis.module.security_group.aws_security_group_rule.default["ingress-tcp-6379-6379-c992040fb21cf75967d80aa440691f00"] will be created
  + resource "aws_security_group_rule" "default" {
      + cidr_blocks              = [
          + "10.76.0.0/16",
          + "10.60.0.0/16",
        ]
      + description              = "Allow TCP inbound traffic on port 6379 from Convox and the vpc which is hosting this redis."
      + from_port                = 6379
      + id                       = (known after apply)
      + protocol                 = "tcp"
      + security_group_id        = (known after apply)
      + self                     = false
      + source_security_group_id = (known after apply)
      + to_port                  = 6379
      + type                     = "ingress"
    }

Plan: 10 to add, 0 to change, 0 to destroy.

File: terraform/production.tfvars [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Variables for deploying to the production stage
+
+deploy_stage     = "production"
+eks_cluster_name = "production"
+app_name         = "databricks-ai-redis-db-production"
+convox_vpc_cidr  = "10.60.0.0/16"
+
+# Redis variables
+redis_name                    = "databricks-ai-redis-cluster"
```
Comment (rgokhale) [inline terraform/production.tfvars:9]: Please append this with the stage name like "databricks-ai-redis-cluster-production", I think we have a bug right now where the module will try to create the datadog alerts with the same name across staging and production.

File: terraform/staging.tfvars [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Variables for deploying to the staging stage
+
+deploy_stage     = "staging"
+eks_cluster_name = "staging"
+app_name         = "databricks-ai-redis-db-staging"
+convox_vpc_cidr  = "10.50.0.0/16"
+
+# Redis variables
+redis_name                    = "databricks-ai-redis-cluster"
```
Comment (rgokhale) [inline terraform/staging.tfvars:9]: "databricks-ai-redis-cluster-staging" based on the above comment

File: terraform/production.tfvars [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Variables for deploying to the production stage
+
+deploy_stage     = "production"
+eks_cluster_name = "production"
+app_name         = "databricks-ai-redis-db-production"
```
Comment (rgokhale) [inline terraform/production.tfvars:5]: This is the name of the app that will use the redis instance, does the above app exist, if there is no specific app, but a process like `sangam`, then you can add that. This is mostly for adding tags for cost estimation purposes.

**Review by rgokhale — APPROVED** [review]
lgtm! Just 2 small nit pick comments.
Please post the production apply to devops support before running that, if you don't have to create a separate PR.

File: terraform/production.tfvars (hunk 4 above)
Comment (VinothRajasekar) [reply terraform/production.tfvars]: thanks. taken care all your comments.

**Pull request merged**